test/format/ops.c

```diff
@@ -504,7 +504,7 @@ skip_insert:			if (col_update(tinfo,
 		 */
 		if (!insert) {
 			dir = (int)mmrand(&tinfo->rnd, 0, 1);
-			for (np = 0; np < mmrand(&tinfo->rnd, 1, 8); ++np) {
+			for (np = 0; np < mmrand(&tinfo->rnd, 1, 30); ++np) {
```

Comment:
> Were you able to reproduce the issue when run with the old code with this change?

I originally made changes in test/format to try and catch the WT-2307 failure (see WT-2302, #2406), but discarded those changes in favor doing the same checks inside the library. Those original changes never detected the bug.

If I apply these changes to a version of the develop branch without the WT-2307 fixes and run the wtperf split-stress workload, these changes detect the bug.